docs/Changelog.md

```diff
@@ -40,14 +40,14 @@
   * Fixed the ability to download an unrelated attachment from a given scenario
 * [#5522](https://github.com/TouK/nussknacker/pull/5522), [#5519](https://github.com/TouK/nussknacker/pull/5519) Scenario status caching more often
 * [#5505](https://github.com/TouK/nussknacker/pull/5505) Fix: anonymous user handling regression
-* [#5438](https://github.com/TouK/nussknacker/pull/5438) [#5495](https://github.com/TouK/nussknacker/pull/5495) Improvement in DeploymentManager API:
-    * Alignment in the api of primary (deploy/cancel) actions and the experimental api of custom actions.
+* [#5438](https://github.com/TouK/nussknacker/pull/5438) [#5495](https://github.com/TouK/nussknacker/pull/5495) [#5610](https://github.com/TouK/nussknacker/pull/5610) Improvement in DeploymentManager API: Alignment in the api of primary (deploy/cancel) actions and the experimental api of custom actions.
```

Comment:
The target branch is staging so it should be rather in 1.15.0. Am I right?

Reply:
Fixed.

Reply:
I can't see this "fix". It is sill in 1.14 section.